src/lib.rs

```diff
@@ -1024,17 +1136,20 @@ mod universe_tests {
     use super::*;
 
     fn generate_test_universe_with_default_params() -> Universe {
```

Comment:
Cool, I didn't know you could have helper functions in the testing module.

Reply:
Yeah it's really nifty since it's simply a special module.